File: app/src/main/java/com/BlizzardArmory/diablo/Character/DyeColor.java
package com.BlizzardArmory.diablo.Character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang.builder.ToStringBuilder;

public class DyeColor {

    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("icon")
    @Expose
    private String icon;
    @SerializedName("tooltipParams")
    @Expose
    private String tooltipParams;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public String getTooltipParams() {
        return tooltipParams;
    }

    public void setTooltipParams(String tooltipParams) {
        this.tooltipParams = tooltipParams;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("id", id).append("name", name).append("icon", icon).append("tooltipParams", tooltipParams).toString();
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/character/items/DyeColor.kt
package com.BlizzardArmory.model.diablo.character.items

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Dye color.
 */
@Keep
data class DyeColor(

    @SerializedName("id")
    var id: String,

    @SerializedName("name")
    var name: String,

    @SerializedName("icon")
    var icon: String,

    @SerializedName("tooltipParams")
    var tooltipParams: String

)